core/interfaces/i_keyboard_accessible.js

```diff
@@ -15,7 +15,7 @@ goog.module('Blockly.IKeyboardAccessible');
 goog.module.declareLegacyNamespace();
 
 /* eslint-disable-next-line no-unused-vars */
-const {KeyboardShortcut} = goog.requireType('Blockly.ShortcutRegistry');
+const ShortcutRegistry = goog.requireType('Blockly.ShortcutRegistry');
```

Comment:
This de-destructuring probably wasn't *really* necessary: since this is a `requireType` (and not just `require`) it can't be necessary to stub/mock this dependency in tests—and the name `KeyboardShortcut` seems adequately unambiguous on its own.

Reply:
The compiler yelled without it, I believe about not being able to destructure a default export.

Reply:
Oh, yeah.  Right.  I forgot it has _opinions_ about this sort of thing.

Moving away from default exports will help solve this.